examples/Bookshelf/androidApp/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+plugins {
+    id("com.android.application")
+    kotlin("android")
+}
+
+val compose_version = "1.0.0-beta08"
+
+repositories {
+    google()
+    mavenCentral()
+}
+
+dependencies {
+    implementation(project(":shared"))
+    implementation("com.google.android.material:material:1.3.0")
+    implementation("androidx.appcompat:appcompat:1.2.0")
+    implementation("androidx.constraintlayout:constraintlayout:2.0.4")
+    implementation("androidx.compose.compiler:compiler:$compose_version")
+    compileOnly("io.realm.kotlin:library:0.1.0")
```

Comment:
If you use the model class in the Android project you will need to resolve `io.realm.RealmObject` otherwise you'll have a compiler error
```
e: Supertypes of the following classes cannot be resolved. Please make sure you have the required dependencies in the classpath:
    class io.realm.sample.bookshelf.model.Book, unresolved supertypes: io.realm.RealmObject
```